ui-components/src/headers/Heading.tsx

```diff
@@ -16,6 +16,7 @@ const HeaderStyleMap = {
   tableSubheader: "table-subheader",
   sidebarHeader: "text-caps-underline",
   categoryHeader: "category-header",
+  sidebarSubHeader: "text-caps-tiny",
```

Comment:
I apologize that I missed something on the recent PR where you added `style` as a prop for `Heading`…that should probably be called `styleType` for (a) consistency with `Button` which uses `styleType`, and (b) the reasoning for that nomenclature which is that `style` implies changing the HTML style attribute of an element. In this case that's not what's happening but rather it's just an enum of class names. If I didn't look closely at the docs/usage of this component I would think `style` is about defining inline styles for an element rather than providing an enum key. Let me know if you think it's worth making a (possibly) breaking change to rename the prop.

Reply:
Definitely think we should do this, but I'm going to spin off a separate ticket since it would create changes in a bunch of files 👍